Wordle.Api/Services/WordService.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Wordle.Api.Data;
+using System.Linq;
+using static Wordle.Api.Data.Game;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Concurrent;
+using Wordle.Api.Dtos;
+
+namespace Wordle.Api.Services;
+
+public class WordService
+{
+    private readonly AppDbContext _context;
+
+    public WordService(AppDbContext context)
+    {
+        _context = context;
+    }
+
+    public IEnumerable<Word> GetWordList(int wordsPerPage, int pageNum, string wordFilter)
+    {
+
+        if (String.IsNullOrWhiteSpace(wordFilter))
+        {
+            var result = _context.Words
+                .Skip((pageNum - 1) * wordsPerPage)
+                .Take(wordsPerPage)
+                .OrderBy(x => x.Value);
```

Comment:
I recommend doing the order by before the take and possibly before the skip. this will make things ordered from the get go.

Reply:
Running the order by first lead to delays of 10+ seconds! Running it before Take would sort ~12k - (pageNum)*wordsPerPage values that would not be required for a call.